lib/tasks/helpers/config.html.erb

```diff
@@ -42,9 +42,9 @@ In other words, environment variables override all other configuration settings
 
 ## View and edit config file options [#Edit]
 
-The Ruby agent's `newrelic.yml` is a standard YAML configuration file. It typically includes a `Defaults` section at the top, plus sections below for each application environment; for example, `Development`, `Testing`, and `Production`.
+The Ruby agent's `newrelic.yml` is a standard YAML configuration file. It typically includes a `Defaults` section at the top, plus sections below for each application environment (for example, `Development`, `Staging`, and `Production`).
```

Comment:
An improvement as-is! Since we're listing 3 of the 4 envs, thoughts on removing the 'for example' and calling them all out? 

(`Development`, `Test`, `Staging`, and `Production`)

Reply:
Great catch! Thank you! 
```suggestion
The Ruby agent's `newrelic.yml` is a standard YAML configuration file. It typically includes a `Defaults` section at the top, plus sections below for each application environment (`Development`, `Test`, `Staging`, and `Production`).
```